packages/react-native-codegen/src/parsers/flow/index.js

```diff
@@ -15,52 +15,32 @@ import type {SchemaType} from '../../CodegenSchema.js';
 // $FlowFixMe[untyped-import] there's no flowtype flow-parser
 const flowParser = require('flow-parser');
 const fs = require('fs');
-const {visit, buildSchemaFromConfigType} = require('../utils');
+const {buildSchemaFromConfigType, getConfigType} = require('../utils.js');
 const {buildComponentSchema} = require('./components');
 const {wrapComponentSchema} = require('./components/schema');
 const {buildModuleSchema} = require('./modules');
 const {isModuleRegistryCall} = require('./utils');
 
-function getConfigType(
-  // TODO(T71778680): Flow-type this node.
-  ast: $FlowFixMe,
-): 'module' | 'component' | 'none' {
-  let isComponent = false;
-  let isModule = false;
-
-  visit(ast, {
-    CallExpression(node) {
+function Visitor(infoMap: {isComponent: boolean, isModule: boolean}) {
```

Comment:
nit: the `V` is capital on purpose because this is a type? Or can we have it lowercase like all the other functions?

Reply:
Yes, I kept capital on purpose. Should i change it lowercase?